examples/aws_provider/test/integration/example/controls/local_security_group.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-# Copyright 2016 New Context Services, Inc.
+# Copyright 2016-2017 New Context Services, Inc.
```

Comment:
Adding end year is not necessary for a copyright notice. Copyrights are effective of the begin date onwards.